docs/source/install/k8s_ha.rst

```diff
@@ -312,7 +377,7 @@ Feedback Needed!
 ----------------
 As this deployment method new and beta is in progress, we ask you to try it and provide your feedback via
 bug reports, ideas, feature or pull requests in `StackStorm/stackstorm-ha <https://github.com/StackStorm/stackstorm-ha>`_,
-and ecourage discussions in `Slack <https://stackstorm.com/community-signup>`_ ``#docker`` channel or write us an email.
+and ecourage discussions in `Slack <https://stackstorm.com/community-signup>`_ ``#k8s`` channel or write us an email.
```

Comment:
Typo.  "encourage".

Reply:
Also OK to drop "write us an email" part.